console/line.go

```diff
@@ -423,6 +422,16 @@ func (c *Component) graphLineHandlerFunc(gc *gin.Context) {
 	gc.JSON(http.StatusOK, output)
 }
 
+func getMax(values []int) int {
+	valMax := values[0]
+	for _, v := range values {
+		if v > valMax {
+			valMax = v
+		}
+	}
+	return valMax
+}
```

Comment:
There is also `slices.Max()`.

Reply:
Removed the `getMax` function, no longer useful